BGAnimations/ScreenSelectMusic overlay/PaneDisplay.lua

```diff
@@ -20,34 +20,20 @@ local GetSongAndSteps = function(player)
 end
 
 -- -----------------------------------------------------------------------
--- requires a profile (machine or player) as an argument
--- returns formatted strings for player tag (from ScreenNameEntry) and PercentScore
-
-local GetScoreAndName = function(profile, SongOrCourse, StepsOrTrail)
+local GetLocalScore = function(profile, SongOrCourse, StepsOrTrail)
 	-- if we don't have everything we need, return empty strings
```

Comment:
We're returning nil now instead of empty strings so update comment.

Reply:
done!